workspace/src/utility.js

```diff
@@ -1,24 +1,24 @@
-export function makeOpIns (pos, value) {
+export function makeOpIns(pos, value) {
```

Comment:
It it is only used in test we don't have to change it. It does feel a bit redundant though.